frontend/src/metabase/home/components/Activity.jsx

```diff
@@ -168,14 +168,12 @@ export default class Activity extends Component {
           );
         } else {
           description.summary =
-            item.model === "dataset" ? t`saved a dataset` : t`saved a question`;
+            item.model === "dataset" ? t`saved a model` : t`saved a question`;
         }
         break;
       case "card-delete":
         description.summary =
-          item.model === "dataset"
-            ? t`deleted a dataset`
-            : t`deleted a question`;
+          item.model === "dataset" ? t`deleted a model` : t`deleted a question`;
```

Comment:
Not a blocker but this text is curious since you can only archive a question/model, not delete one.

Reply:
@mazameli That was my first thought as well. Maybe it's something from the old ages OR...

Maybe it's there because you actually can delete something from inside the archive.
![image](https://user-images.githubusercontent.com/31325167/149854381-9d467e1e-9a4a-4030-83f9-c00202d6e98a.png)

Reply:
But interestingly, there is a bug there and the deleted model is not recognized as such - it still says "Deleted a question"
![image](https://user-images.githubusercontent.com/31325167/149854621-6f3dc470-529b-49e2-ba4e-c1bea5310f53.png)